### ISSUE TITLE
StoreStock and Pricing

### ISSUE THREAD

**Issue by farooqaaa** [django-oscar/django-oscar-stores#157]
Why doesn't the StoreStock model have a price field? Is it left out intentionally?

Also, I think it would be a good idea to extend Oscar's StockRecord model. It's not possible to extend it cleanly because it has a partner field and if you extend it, you will have to keep that field.

**Comment by solarissmoke** [comment]
> Why doesn't the StoreStock model have a price field? Is it left out intentionally?

Yes - this isn't really intended to be a full stock management system for a store. `StoreStock` is intended just to keep track of whether an item is available at a given store - so that you could, for example, show customers the nearest store where it is available. I think attaching in-store pricing to this is beyond the scope of this package (although you could extend it to do something like that if you wanted to).
